.github/workflows/pretest.yml

```
@@ -0,0 +1,23 @@
+name: "Pre-review Tests"
+
+on: [push, pull_request]
+
+jobs:
+  pretest:
+    runs-on: ubuntu-20.04
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: setup python
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.7'
+
+    - name: install
+      run: |
+        pip install pygithub pysen mypy
```

I think it's better to use mypy version pinned by pysen.
```suggestion
        pip install pygithub 'pysen[lint]'
```